src/core/src/sim/md/controls.rs

```diff
@@ -104,9 +105,27 @@ impl Control for BerendsenThermostat {
 impl Thermostat for BerendsenThermostat {}
 
 /******************************************************************************/
+
+impl<T> Control for Alternator<T> where T: Control {
+
+    fn control(&mut self, system: &mut System) {
+        if self.can_run() {
+            self.base_mut().control(system)
+        }
+    }
+}
+
 /// Remove global translation from the system
 pub struct RemoveTranslation;
 
+impl RemoveTranslation {
+    /// Create a new `RemoveTranslation"` control, that will
+    /// run every `every` step of the simulation.
+    pub fn new() -> RemoveTranslation {
```

Comment:
We probably can. However this is easier for backwards compatibility: if we ever need to add some fields to the struct, using only `RemoveTranslation` will break, while `RemoveTranslation::new()` could set some reasonable defaults.